proposals/20200504-falco-artifacts-scope.md

```diff
@@ -0,0 +1,191 @@
+
+_If a container image is for development purproses it MUST have the `dev-` prefix._ 
+
+_If a container image is used to install something, then exit it MUST have the `-install` suffix.*_
+
+---
+
+ - falcosecurity/falco:TAG
+     - Runs `falco` userspace only
+ - falcosecurity/privileged-driver-install:TAG
+     - Runs `falco-driver-loader` and exit
+ - falcosecurity/priviliged-driver-install-falco 
+     - first runs `privileged-driver-install` then runs `falco`
+ - falcosecurity/priviliged-host-systemd-driver-falco:TAG
+     - Detects arch and runs new bash script:
+         - Installs Falco with packages (deb, rpm, src)
+         - Configures with systemd and Unix Domain Socket [#858](https://github.com/falcosecurity/falco/issues/858)
+ - falcosecurity/dev-tester:TAG 
+     - Runs the falco test suite
+ - falcosecurity/dev-builder:TAG
+     -  Contains falco tool chain
+
+
+# Falco Project Evolution
+
+We will modeling a loosely defined adoption of the Kubernetes and CNCF incubator efforts.
+
+The criteria will remain loose, and tighten as needed at the discretion of the Falco open source community.
+
+### contrib
+
+"_Sandbox level_"
+
+This new [contrib](https://github.com/falcosecurity/contrib) repository will be equivalent to the `Falco Sandbox` and serves as a place for the community to `test-drive` ideas/projects/code.
+
+### repository
+
+"_Incubating level_" projects such as [falco-exporter](https://github.com/falco-exporter) can be promoted from `contrib` to their own repository. 
+
+This is done as needed, and can best be measured by the need to cut a release and use the github release features. Again, this is at the discretion of the Falco open source community.
+
+### official support
+
+As the need for a project grows, it can ultimately achieve the highest and most coveted status within The Falco Project. "_Offical support_."
+
+These artifacts will be ammended to the ones listed above, and will become a part of the official Falco release process.
+
+# Action
+
+For each item, ask if this already exists. If so we need to rename, and update it to match this new convention.
+
+    
+### Action Items
+
+Here are SOME of the items that would need to be done for example:
+
+ - Rename `stable` image to `privileged-driver-install-falco`
+ - Rename `bin` package to `falco-driver-full-x86.tar.gz`
+ - Rename `slim` image to `falco`
+ - Documentation in all packages with `INSTALL` file.
+ - Move everything else to contrib
+     - Move [/integrations](https://github.com/falcosecurity/falco/tree/master/integrations) to contrib
+     - Move [/examples](https://github.com/falcosecurity/falco/tree/master/examples) to contrib
+     - Old docker files
+     - Old documentation
+
+### Documentation
+
+Update documentation in [falco-website#184](https://github.com/falcosecurity/falco-website/pull/184)
+
+### Adjusting projects
+
+ - Helm chart documentation to be moved to `contrib`
```

Comment:
The helm chart is already outside https://github.com/helm/charts/tree/master/stable/falco